_posts/2023-04-04-flux-pmix-0.3.0.md

```diff
@@ -0,0 +1,33 @@
+---
+title: flux-pmix v0.3.0
+date: 2023-04-04 00:00:00
+author: "flux-framework"
+categories: 'release'
+version: 0.3.0
+download: https://github.com/flux-framework/flux-pmix/releases/download/v0.3.0/flux-pmix-0.3.0.tar.gz
+---
+
+Download from GitHub [here]({{ page.download }})
+
+# Release Notes
+
+flux-pmix version 0.3.0 - 2023-04-04
+------------------------------------
+
+## New Features
+
+ * support flux run -o pmi=pmix (#81)

```

Comment:
These numbers aren't being parsed - need to look into this.

Reply:
okay this doesn't match the expected format - the release notes usually link directly to a NEWS.md (and it wasn't found here). Either we update the release notes to coincide with the others, or we label this with a different function parser. Either way / whatever we decide, we need to be consistent moving forward. Thoughts @grondo @cmoussa1 ?

Reply:
Ah, that wasn't intentionally different.  Let me fix.

Reply:
https://github.com/flux-framework/flux-pmix/releases/tag/v0.3.0

Better?

Reply:
Yep let’s try again!